composer.json

```diff
@@ -14,7 +14,8 @@
         "guzzlehttp/promises": "^1.4||^2.0",
         "guzzlehttp/psr7": "^2.0",
         "google/common-protos": "^4.4",
-        "google/longrunning": "~0.2"
+        "google/longrunning": "~0.2",
+        "ramsey/uuid": "^4.0"
```

Comment:
This dep requires `ext-json`, but apparently so does guzzle, so it shouldn't introduce any dependency errors